<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Viewer</class>
 <widget class="QWidget" name="Viewer">
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="toolTip">
   <string>Right-click on photo to show actions</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>194</width>
        <height>94</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionFitToWindow">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/fit_to_window</normaloff>:/images/fit_to_window</iconset>
   </property>
   <property name="text">
    <string>Fit to window</string>
   </property>
  </action>
  <action name="actionShowFullImage">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/full_image</normaloff>:/images/full_image</iconset>
   </property>
   <property name="text">
    <string>Show full image</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/zoom_in</normaloff>:/images/zoom_in</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/zoom_out</normaloff>:/images/zoom_out</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionRotateClockwise">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/clockwise</normaloff>:/images/clockwise</iconset>
   </property>
   <property name="text">
    <string>Rotate clockwise</string>
   </property>
  </action>
  <action name="actionRotateCClockwise">
   <property name="icon">
    <iconset resource="../res/images.qrc">
     <normaloff>:/images/cclockwise</normaloff>:/images/cclockwise</iconset>
   </property>
   <property name="text">
    <string>Rotate c. clockwise</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/images.qrc"/>
 </resources>
 <connections/>
</ui>
